coalib/bears/requirements/JuliaRequirement.py

```diff
@@ -0,0 +1,31 @@
+from coalib.bears.requirements.PackageRequirement import PackageRequirement
+
+
+class JuliaRequirement(PackageRequirement):
+    """
+    This class is a subclass of ``PackageRequirement``, and helps specifying
+    requirements from ``julia``, without using the manager name.
+    """
+
+    def __init__(self, package, version="", flag=""):
+        """
+        Constructs a new ``JuliaRequirement``, using the ``PackageRequirement``
+        constructor.
+
+        >>> pr = JuliaRequirement('setuptools', '19.2', '-e')
+        >>> pr.manager
+        'julia'
+        >>> pr.package
+        'setuptools'
+        >>> pr.version
+        '19.2'
+        >>> pr.flag
+        '-e'
+
+        :param package: A string with the name of the package to be installed.
+        :param version: A version string. Leave empty to specify latest version.
+        :param flag:    A string that specifies any additional flags, that
+                        would be used with ``flag``.
```

Comment:
Because this is metadata, when i'm going to write the function that grabs information and writes down the commands automatically, it shuold be aware of flags and stuff.